andy_api/run_dev_demo.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+# Change this to the proper path
+# # For Bricson:
+# ABSOLUTE_PATH_TO_KEY="C:\Users\selen\Desktop\ChessToSpeech\chess-to-speech\andy_api\service_key_file.json"
+# For Zach:
+ABSOLUTE_PATH_TO_KEY="/home/natestull/Documents/School/CIS4930 - Special Topics in CISE: Spoken Dialogue Systems/Project/develop/chess-to-speech/andy_api/api/service_key_file.json"
```

Comment:
Instead of deleting the path to my key, add another line with a comment like `For Nate:` with the path to your key, so that we can all swap between them easily.